pkg/cache/clusterqueue.go

```diff
@@ -315,6 +347,7 @@ func (c *clusterQueue) UpdateWithFlavors(flavors map[kueue.ResourceFlavorReferen
 
 func (c *clusterQueue) updateLabelKeys(flavors map[kueue.ResourceFlavorReference]*kueue.ResourceFlavor) {
```

Comment:
Could you extend the unit testing?
https://github.com/kubernetes-sigs/kueue/blob/5203ac9eb69226a1ae3b6ba6d361eaed7ef3dc8c/pkg/cache/clusterqueue_test.go#L38

Reply:
I think the unit tests here aren't very useful as suggested in another comment. I would prefer to cover the `inactiveReason` function as in the other comment.